_posts/bulk-editing-garmin-data.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+author: Mauro Servienti
+title: "They'll probably make it anyway: bulk editing Garmin data"
+synopsis: "Users have needs. They want to get things done when using software. If what they have is not satisfactory, they'll get creative."
+header_image: /img/posts/bulk-editing-garmin-data/header.jpg
+tags:
+- personal
+---
+
+I'm a Garmin user, as I said [multiple times](https://milestone.topics.it/tags/personal.html). To track and navigate during outdoor bike rides, I use a Garmin Edge 830. When the weather is terrible, I ride indoor using an old bike and an Elite Suito trainer. My friend [Sandro](https://www.rizzetto.com/), who lives in a beautiful area, makes fun of me. Take a look at [some of his photos in the gallery](https://www.rizzetto.com/photo/default.aspx)...he has a point!
+
+Enough talking! Garmin Connect, the application used to review all the stats collected by Garmin devices, offers a nice Gears feature. It is a database of all the gear someone uses during sports activities. For example, equipment like running shoes or bikes are defined in the Gears database and later associated with performed activities. When adding gears, we can specify one of them as the default that Garmin will automatically use for future activities.
+
+Bike riding is my primary activity alongside swimming, and the default gear is my road bike. The issue is that when I'm training indoor using the trainer, activities get associated with the road bike. Garmin should introduce a feature that allows associating default gears to activity types instead of having a generic default gear for every activity. Searching through Garmin public forums, the feature has quite a few requests. The oldest I've found is from 2017. That only confirms that Garmin is sort of deaf to users' feedback.
+
+Let's imagine for a second that it's an incredibly complex feature. In the end, I have no idea what the data and services structure is behind the scene. It could be that it's hard to implement.

```

Comment:
```suggestion
Let's imagine for a second that it's an incredibly complex feature. In the end, I have no idea what the data and services structure is behind the scenes. It could be that it's hard to implement.
```